build.gradle

```diff
@@ -51,256 +51,270 @@ minecraft {
     makeObfSourceJar = false // an Srg named sources jar is made by default. uncomment this to disable.
 }
 
+repositories {
+    maven {
+        //Everything
+        name = 'curseforge'
+        url = 'https://minecraft.curseforge.com/api/maven/'
+    }
+}
+
+dependencies {
+    compileOnly "player-api:PlayerAPI:1.10.2:1.1"
+    compileOnly "applied-energistics-2:appliedenergistics2-rv4:stable:1"
+    compileOnly "biomes-o-plenty:BiomesOPlenty-1.10.2:5.0.0.2236:universal"
+    compileOnly "not-enough-items-1-8:NotEnoughItems-1.10.2:2.1.3.220:universal"
+    compileOnly "codechicken-core-1-8:CodeChickenCore-1.10.2:2.4.1.102:universal"
+    compileOnly "codechicken-lib-1-8:CodeChickenLib-1.10.2:2.5.9.320:universal"
+    compileOnly "cofhcore:CoFHCore-1.10.2:4.1.12.17:universal"
+    compileOnly "actually-additions:ActuallyAdditions:1.10.2:r105"
+    compileOnly "industrial-craft:industrialcraft-2:2.6.252:ex110"
+    compileOnly "mekanism:Mekanism:1.10.2:9.2.4.103"
+    compileOnly "jei:jei_1.10.2:3.14.7.420"
+    compileOnly "not-enough-items-1-8:NotEnoughItems-1.10.2:2.1.3.220:universal"
+    //compileOnly "minefactory-reloaded:MineFactoryReloaded-[1.10.2]2.9.0B1:226" //No .pom file (errors)
+
+}
+
 // Keep a list of dependencies files/directories for easy jar packing
-ext.dependencies = ['codechicken/**',
-					'**/nei_at.cfg',
-					'api/**',
-					'appeng/**',
-					'biomesoplenty/**',
-					'buildcraft/**',
-					'cofh/**',
-                                        'de/**',
-					'ic2/**',
-					'mekanism/**',
-					'mezz/**',
-					'powercrystals/**']
-					
+ext.dependencies = ['powercrystals/**']
```

Comment:
MFR doesn't have a .pom file on the CurseForge maven, causing errors. This means that `powercrystals/**` still must exist or your workspace won't have MFR.